tests/integration/express-server.spec.ts

```diff
@@ -24,395 +24,409 @@ import {
 const basePath = '/v1';
 
 describe('Express Server', () => {
-  it('can handle get request to root controller`s path', () => {
-    return verifyGetRequest(basePath + '/', (_err, res) => {
-      const model = res.body as TestModel;
-      expect(model.id).to.equal(1);
+  describe('RootController', () => {
```

Comment:
Even with whitespace hidden, the changes in this file are hard to verify. Why do we need to change the express server specs in this PR?

Reply:
When I cover whole tests in express-server.spec, I also grouped these test cases with different describe.